src/components/Footer/Footer.css

```diff
@@ -1,13 +1,42 @@
 footer {
+  width: 100%;
   display: flex;
   justify-content: space-around;
-  width: 100%;
-  margin: 5% 5% 0% 5%;
-  border-top: 1px solid #2c3e50;
+  background-color: rgba(245, 245, 245, 0.90);
+  color: #2c3e50;
+  border-radius: .35rem;
+  margin: 5% 0% 0% 0%;
+}
+
+a {
+  color: #2c3e50;
+  font-family: Avenir, Helvetica, Arial, sans-serif;
+  text-decoration: underline;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+a:hover {
+  font-style: italic;
+  cursor: pointer;
 }
 
 @media screen and (max-width: 768px) {
   footer {
+    width: 95%;
     flex-direction: column;
+    padding: 1rem;
+  }
+
+  p{
```

Comment:
Nitpicky, but this is missing a space before the bracket!

Reply:
thanks, nice catch